lib/handlers/get.js

```diff
@@ -60,9 +60,17 @@ function handler (req, res, next) {
       return next(err)
     }
 
+    // Redirect to container with / if missing
+    if (container && path.lastIndexOf('/') !== path.length - 1) {
+      debug(contentType)
+      return res.redirect(301, _path.join(path, '/'))
```

Comment:
do we really want to redirect every time? I am not sure I understand why this should _always_ happen